django/contrib/admin/tests.py

```diff
@@ -231,3 +233,24 @@ def has_css_class(self, selector, klass):
             .find(klass)
             != -1
         )
+
+    def take_screenshot(self, name):
+        if not self.screenshots:
+            return
+        window_size = self.selenium.get_window_size()
+        screenshot_specs = [
+            {"name": "desktop", "width": 1920, "height": 1200},
+            {"name": "mobile", "width": 360, "height": 800},
+        ]
+        for spec in screenshot_specs:
+            self.selenium.set_window_size(spec["width"], spec["height"])
```

Comment:
If I'm not mistaken, we don't have screenshot with rtl direction in HTML. I think it will be interesting if we can have that.
I'm not completely familiar with selenium, do you think it's possible at this stage ?

Reply:
It should be possible for sure and it's a good idea, do you think it's similar to mobile where we would want it for every page?

Reply:
I think so, but if it's too many screenshots to have/check on, maybe we could have screenshots on list page and detail page, IMO these are the most significant pages for rtl

Reply:
IMO it's good to have everywhere because while I usually remember to take screens of RTL, I always forget to test it when making further commits.

Reply:
Ok I added logic that you can take the screenshots also in RTL. You can view screenshots here: https://github.com/django/django/actions/runs/5359753461/
It's an attachment named screenshots at the bottom :+1:

Reply:
LGTM 🚀